man/bundle-pristine.ronn

```diff
@@ -8,6 +8,29 @@ bundle-pristine(1) -- Restores installed gems to their pristine condition
 ## DESCRIPTION
 
 `pristine` restores the installed gems in the bundle to their pristine condition using the local gem
-cache from RubyGems. For git gems - a forced checkout will be performed.
+cache from RubyGems.
+For git gems, a forced checkout will be performed.
+`bundle pristine` ignores unpacked files on disk. This command
+utilizes the local `.gem` cache or the gem's git repository
+as if one were installing from scratch.
 
-Note the Bundler gem cannot be restored to its original state with `pristine`.
+Note: the Bundler gem cannot be restored to its original state with `pristine`.
+One also cannot use `bundle pristine` on local gems, because bundler has no
```

Comment:
how about swapping `local gems`  with `gems with a 'path' option in the Gemfile` to make it more clear to the user?